app/tests/reader_studies_tests/test_views.py

```diff
@@ -135,6 +135,43 @@ def test_answer_remove(client):
     assert Answer.objects.filter(creator=r2).count() == 1
 
 
+@pytest.mark.django_db
+def test_answer_remove_ground_truth(client):
```

Comment:
These tests do not check that answers and ground truth for other reader studies are unaffected.